### PR TITLE
added question on network models

### PR DESCRIPTION
**Pull request by PranjalSK03**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by PranjalSK03** [comment]
is there any problem with my question?

**Comment by PranjalSK03** [comment]
@jdwilkin4 
please look on this question too.

File: src/data/information-technology-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const informationTechnologyQuiz = [
+  {
+    Question: "How many layers are there in OSI(Open Systems Interconnection) and (TCP/IP)Transmission Control Protocol/Internet Protocol Models respectively",
```
Comment (jdwilkin4) [inline src/data/information-technology-quiz.ts:3]: Long question usually don't look great on the site. Let's shorten this

```suggestion
    Question: "How many layers are there in the OSI and TCP/IP models respectively?",
```

File: src/data/information-technology-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    Answer:"7,4",
+    Distractor1: "4,7",
+    Distractor2: "7,5",
+    Distractor3: "11,7",
```
Comment (jdwilkin4) [inline src/data/information-technology-quiz.ts:7]:
```suggestion
    Answer:"7 and 4",
    Distractor1: "12 and 7",
    Distractor2: "7 and 5",
    Distractor3: "11 and 7",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by PranjalSK03 — COMMENTED** [review]
made the changes as required.

**Review by PranjalSK03 — COMMENTED** [review]
made final changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**